apollo/src/commonTest/kotlin/io/iohk/atala/prism/apollo/derivation/EdHDKeyTest.kt

```diff
@@ -0,0 +1,23 @@
+package io.iohk.atala.prism.apollo.derivation
+
+import io.iohk.atala.prism.apollo.Platform
+import io.iohk.atala.prism.apollo.utils.decodeHex
+import io.iohk.atala.prism.apollo.utils.toHexString
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class EdHDKeyTest {
+    @Test
+    fun test_derive_m1() {
+        if (!Platform.OS.contains("Android")) {
```

Comment:
can someone explain this? @hamada147 I think I remember you adding it?

Reply:
This was because we needed to make sure that the tests were running only on `JVM` from the `commonTest` and not `Android` as the IDE will think of it as part of the `JVM` tests, which will fail due to not finding the correct binary inside the `JNI` folder. You can copy the same test to the Android tests folder so that it will run on the `Android` environment using the correct `Android` context and not the `JVM` one.

Reply:
done